src/bounding_box.rs

```diff
@@ -84,32 +85,44 @@ impl BoundingBox {
     /// Returns whether a given point is inside (or on the edges) of the bounding box.
     #[inline]
     pub fn is_inside(&self, point: &Point) -> bool {
-        point.x.abs() <= self.top_right.x && point.y.abs() <= self.top_right.y
+        // left.x <= point.x <= right.x
+        let horizonal_ok = (self.top_right.x - self.width() <= point.x ) && (point.x <= self.top_right.x);
+        // bottom.y <= point.y <= top.y
+        let vertical_ok = (self.top_right.y - self.height() <= point.y) && (point.y <= self.top_right.y);
+
+        horizonal_ok && vertical_ok
     }
 
     // /// Same as inside, but return false if point is on the box edge.
-    // #[inline]
-    // pub fn is_exclusively_inside(&self, point: &Point) -> bool {
-    //     point.x.abs() < self.top_right.x && point.y.abs() < self.top_right.y
-    // }
+    #[inline]
+    pub fn is_exclusively_inside(&self, point: &Point) -> bool {
```

Comment:
I forgot I had left this commented out. I was debating with myself whether I should keep the BoundingBox API public, as that was not the main goal of the library. Do you see a use case to expose it?

Reply:
I would probably keep part of this API public, up to including `which_edge()`, such that users can iterate over their points (or new ones) to deal with special cases such as edges. And also expose both the original and relaxed sites, like in [`Voronoi.sites()`](https://github.com/andreesteve/voronoice/blob/9df72656ed73f7c5541d390e27d082af0c6361a2/src/lib.rs#L178)

This _could_ be used to stitch together neighboring Voronoi-diagrams or other forms of polygon-meshes ( e.g. infinite world generation in games, 2d meshing in FEM/CAM with contact-materials [this might actually require being able to fix boundary points in place while only relaxing inner ones]).

The rest of the function, `intersect_line()`, `project_ray()`, `project_ray_closest()` and `order_points_on_ray`, probably have no use outside of the diagram-generation and can be hidden

Reply:
And sure, I'll write up a few unit tests for this tomorrow!

Reply:
Thank you! I agree it makes sense to keep the ```which_edge``` public.